bin/vcloud-launch

```diff
@@ -2,4 +2,37 @@
 
 require 'vcloud/launcher'
 
-Vcloud::Launcher::Cli.new(ARGV).run
+banner_text = <<EOF
+vcloud-launch takes a configuration describing a vCloud Org,
+and tries to make it a reality.
+
+See https://github.com/gds-operations/vcloud-tools for more info
+EOF
+
+opts = Trollop::options do
+  banner banner_text
+
+  opt :config_file,       "Configuration file",    required: true
```

Comment:
Discussion from meatspace: This would make the configuration file a command-line option rather than an argument as it is currently.

My preference is to keep it as an argument; that would seem more idiomatic for Unix tools. Also, the configuration's file importance as a required argument and the location where the machines to be launched are defined would suggest to me it makes sense to keep as an argument.